lib/dogma/rules/final_newline.ex

```diff
@@ -8,7 +8,9 @@ defmodule Dogma.Rules.FinalNewline do
   alias Dogma.Script
   alias Dogma.Error
 
-  def test(script) do
+  def test(script), do: test(script, [])
```

Comment:
We don't need this if `test/2` is defined with a default argument in the second position.